src/components/Overview/index.js

```diff
@@ -0,0 +1,338 @@
+            actionTitle={item.actionTitle}
+            action={item.action}
+            content={item.content}
+            learnMoreLink={item.learnMoreLink}
+          />
+        ))}
+      </DataList>
+
+      <Title headingLevel="h2" className="pf-u-mb-md">
+        {intl.formatMessage({
+          id: 'integrations.overview.titleRecommendedContent',
+          defaultMessage: 'Recommended content',
+        })}
+      </Title>
+      <Table aria-label="Recommended content" className="pf-u-mb-lg">
+        <Tbody>
+          <Tr>
+            <Td>
+              {intl.formatMessage({
+                id: 'integrations.overview.recommendedContentTableTitle1',
+                defaultMessage: 'Configuring notifications and integrations',
+              })}
+            </Td>
+            <Td>
+              <Label color="orange">
+                {intl.formatMessage({
+                  id: 'integrations.overview.recommendedContentTableLabel1',
+                  defaultMessage: 'Documentation',
+                })}
+              </Label>
+            </Td>
+            <Td className="pf-v5-u-text-align-right">
+              <a
+                href={VIEW_DOCUMENTATION}
+                onClick={(e) => {
+                  e.preventDefault();
+                  window.open(VIEW_DOCUMENTATION, '_blank');
+                }}
+              >
+                {intl.formatMessage({
+                  id: 'integrations.overview.recommendedContentTableLink1',
+                  defaultMessage: 'View documentation',
+                })}{' '}
+                <ExternalLinkAltIcon />
+              </a>
+            </Td>
+          </Tr>
+          <Tr>
+            <Td>
+              {intl.formatMessage({
+                id: 'integrations.overview.recommendedContentTableTitle2',
+                defaultMessage: 'Configuring console notifications in Slack',
+              })}
+            </Td>
+            <Td>
+              <Label color="green">
+                {intl.formatMessage({
+                  id: 'integrations.overview.recommendedContentTableLabel2',
+                  defaultMessage: 'Quick start',
+                })}
+              </Label>
+            </Td>
+            <Td className="pf-v5-u-text-align-right">
+              <Button variant="link" onClick={handleActivateQuickstart} isInline>
+                {intl.formatMessage({
+                  id: 'integrations.overview.recommendedContentTableLink2',
+                  defaultMessage: 'Begin Quick start',
+                })}{' '}
+                <ArrowRightIcon />
+              </Button>
+            </Td>
+          </Tr>
+        </Tbody>
+      </Table>
+
+      <a href={`${isBeta() ? '/preview' : ''}/settings/learning-resources`} className="pf-u-mb-lg">
+        {intl.formatMessage({
+          id: 'integrations.overview.viewAllLearningResources',
+          defaultMessage: 'View all Settings learning resources',
+        })}
+      </a>
```

Comment:
Can we use PF link button instead?
Also with the killed `/preview`, we may not need the changes in the URL

Reply:
Fixed